pom.xml

```diff
@@ -43,6 +43,12 @@
             <version>18.0.0</version>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>org.hamcrest</groupId>
```

Comment:
there's already `assertj` library for asserting in tests. `hamcrest` is for the same goal. It's better to have a single way for asserting in classpath

Reply:
fixed